Subject: [PATCH] libjpeg-turbo: Enable fast DCT scaling

Only really helps for images with a quality higher than 90, but gives about 15% speed increse

diff --git a/Android.mk b/Android.mk
index 2801805..aade9b8 100644
--- a/Android.mk
+++ b/Android.mk
@@ -66,7 +66,7 @@ LOCAL_C_INCLUDES_x86_64 += $(LOCAL_PATH)/simd
 LOCAL_SRC_FILES_mips += jsimd_none.c
 LOCAL_SRC_FILES_mips64 += jsimd_none.c
 
-LOCAL_CFLAGS += -O3 -fstrict-aliasing
+LOCAL_CFLAGS += -O3 -fstrict-aliasing -DJDCT_DEFAULT=JDCT_IFAST
 LOCAL_CFLAGS += -Wno-unused-parameter
 LOCAL_EXPORT_C_INCLUDE_DIRS := $(LOCAL_PATH)
 
